cnf-certification-test/operator/suite.go

```diff
@@ -79,11 +79,18 @@ func testOperatorInstallationWithoutPrivileges(env *provider.TestEnvironment) {
 	var compliantObjects []*testhelper.ReportObject
 	var nonCompliantObjects []*testhelper.ReportObject
 	for i := range env.Operators {
-		csv := env.Operators[i].Csv
+		operator := env.Operators[i]
+		csv := operator.Csv
 		clusterPermissions := csv.Spec.InstallStrategy.StrategySpec.ClusterPermissions
 		if len(clusterPermissions) == 0 {
-			logrus.Debugf("No clusterPermissions found in %s", env.Operators[i])
-			compliantObjects = append(compliantObjects, testhelper.NewOperatorReportObject(env.Operators[i].Namespace, env.Operators[i].Name, "Operator has no privileges on cluster resources", true))
+			logrus.Debugf("No clusterPermissions found in %s", operator)
+			compliantObjects = append(compliantObjects, testhelper.NewOperatorReportObject(operator.Namespace, operator.Name, "Operator has no privileges on cluster resources", true))
+			continue
+		}
+
+		if operator.IsClusterWide {
```

Comment:
Shouldn't we add also a check on the all namepace install mode here? For example, with kiali-ossm operator, isClusterWide is false, but installMode only includes All namespaces:
![image](https://github.com/test-network-function/cnf-certification-test/assets/86730676/d66db435-da99-4fae-9d41-b1578e1d381a)

Reply:
I've tested it with the kiali-ossm operator and the isClusterWide flag was correctly set to true. ¿Have you included in the tnf_config.yaml the name of the namespace where the subscription of the kiali-ossm operator was created to install it (openshift-operators)?